### ISSUE TITLE
Running tox -e deps is senstive to the python version while updating nightly scipp release.

### ISSUE THREAD

**Issue by MridulS** [scipp/copier_template#163]
Running `tox -e deps` in a python env which doesn't sync with `python-version-ci` will lead to problems where the CI is running a different python in the runner vs the scipp nightly release.

**Comment by SimonHeybrock** [comment]
Yes, that appears to be how tox works. Is there a way of preventing that?

**Comment by MridulS** [comment]
We can enforce that tox -e deps also syncs up `python-version-ci` so atleast we can catch these issues.

Or the CI itself should be running the tox action and creating a new PR :D

**Comment by jl-wynen** [comment]
I think selecting a specific Python version in the `deps` env would be fine. It would prevent running the update with an incorrect version.
